### ISSUE TITLE
Chameleon: no clusters to merge

### ISSUE THREAD

**Issue by deric** [deric/clueminer#5]
```java
java.lang.NullPointerException
	at org.clueminer.chameleon.Merger.createClusters(Merger.java:63)
	at org.clueminer.chameleon.PairMerger.getHierarchy(PairMerger.java:52)
	at org.clueminer.chameleon.Chameleon.hierarchy(Chameleon.java:139)
	at org.clueminer.clustering.ClusteringExecutorCached.hclustRows(ClusteringExecutorCached.java:51)
	at org.clueminer.clustering.ClusteringExecutorCached.clusterRows(ClusteringExecutorCached.java:95)
	at org.clueminer.demo.gui.MatrixWrapper$1.run(MatrixWrapper.java:128)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1423)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2033)
```

in this case it seems that bisection gives `null` result

```java
ArrayList<LinkedList<Node>> partitioningResult = recursiveBisection.partition(datasetMaxPSize, g);
```
@tomasbruna any ideas?

**Comment by tomasbruna** [comment]
Probably happens when the graph is smaller than maximum partition size. Should by fixed in 5b8275ef58293430615039e7d47347c2b31ea135

**Comment by deric** [comment]
ok, thanks.
